resources/qml/Preferences/MachinesPage.qml

```diff
@@ -109,6 +109,10 @@ UM.ManagementPage
         UM.Dialog
         {
             id: actionDialog
+            //NOTE:(RenShen): 给定窗体初大小始化值，避免文字堆叠在一起
```

Comment:
Please remove this comment, attribution is already visible in GitHub (and Git in general). We also want to keep all comments in English.